schema/deploy/tables/attachment.sql

```diff
@@ -1,12 +1,19 @@
 -- Deploy cif:tables/attachment to pg
-
+-- requires: tables/project_status
+-- requires: tables/project
 begin;
 
 create table cif.attachment
 (
   id integer primary key generated always as identity,
   file uuid,
-  file_name varchar(1000)
+  description varchar(10000),
+  file_name varchar(1000),
+  file_type varchar(100),
+  file_size varchar(100),
+  cif_user_id integer not null references cif.cif_user(id),
```

Comment:
the `upsert_timestamp_columns` below inserts a `created_by` column, so this is not needed